static_src/actions/app_actions.js

```diff
@@ -54,27 +74,35 @@ export default {
       appGuid,
       app
     });
+
+    return Promise.resolve(app);
```

Comment:
what's the pattern of what promise data should be returned in actions? So if `receivedAppStats` just took an `appGuid` should that be returned or nothing? It'd be helpful to know the convention.